### PR TITLE
Task 1 completed

### PR DESCRIPTION
**Pull request by GeViN-Gl**

### CODE REVIEW

File: src/components/Courses/component/CourseCard/CourseCard.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	return namesStr;
+};
+
+const CourseCard = ({ course }) => {
+	const { title, description, creationDate, duration, authors } = course;
+
+	const { authorsList } = useContext(AuthorsContext);
+
+	if (!authorsList) {
+		console.log(authorsList);
```
Comment (KitTheProgrammer) [inline src/components/Courses/component/CourseCard/CourseCard.jsx]: Don't forget to remove console.log when committing

File: src/components/CreateCourse/components/AddAuthor/AddAuthor.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      <Title>Add author</Title>
+      <AddAuthorMiniform>
+        <Input
+          labelText='Author name'
+          placeholderText='Enter author name...'
+          onChange={inputHandler}
+          value={inputValue}
+        />
+        <Button
+          onClick={(event) => {
```
Comment (KitTheProgrammer) [inline src/components/CreateCourse/components/AddAuthor/AddAuthor.jsx]: This handler should not be inline (too big)

File: src/components/CreateCourse/components/AuthorCard/AuthorCard.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  const {
+    notAddedAuthorList,
+    addedAuthorList,
+    setNotAddedAuthorList,
+    setAddedAuthorList,
+  } = useContext(CreateCourseContext);
+
+  // Define a named function that handles the button click // Render the
+
+  const handleAuthorClick = (event) => {
```
Comment (KitTheProgrammer) [inline src/components/CreateCourse/components/AuthorCard/AuthorCard.jsx]: It will be beneficial to use "useCallback" here

**Comment by GeViN-Gl** [comment]
Test comment

**Review by KitTheProgrammer — APPROVED** [review]

**Comment by KitTheProgrammer** [comment]
20 points

**Pull request merged**